skeleton/backend/src/Backend.hs

```diff
@@ -1,10 +1,22 @@
+{-# LANGUAGE OverloadedStrings #-}
 module Backend where
 
-import Common.Api
-import Frontend
+import qualified Data.ByteString.Char8 as BSC
+
 import qualified Obelisk.Backend as Ob
+import Snap (Snap, writeBS)
+
+import Common.Api (commonStuff)
+import Frontend (frontend)
 
 backend :: IO ()
-backend = Ob.backend Ob.def
+backend = Ob.backend $ Ob.def
   { Ob._backendConfig_head = fst frontend
+  , Ob._backendConfig_routes =
+      [ ("/api", someApi)
+      ]
   }
+
+someApi :: Snap ()
+someApi = do
+  writeBS $ BSC.pack commonStuff
```

Comment:
@ryantrinkle if I understand [your comment](https://github.com/obsidiansystems/obelisk/pull/32#issuecomment-393898442) correctly, this will need changing so that the user does not have to directly import/use `Snap` right? If so, I wonder how that would look ...